ceph/rados/rados_scrub.py

```diff
@@ -278,3 +278,52 @@ def get_scrub_error_count(self):
         return status_out_put["health"]["checks"]["OSD_SCRUB_ERRORS"]["summary"][
             "count"
         ]
+
+    def get_osd_logs(self, init_time, end_time, osd_id):
+        """
+        Method to check the scrub and deep-scrub logs exists or not
+        Args:
+            rados_object: Rados object
+            init_time:  initial time
+            end_time:  End time
+            osd_log_lines: log lines to check in th logs
+
+        Returns:
+        """
+        fsid = self.run_ceph_command(cmd="ceph fsid")["fsid"]
+        host = self.fetch_host_node(daemon_type="osd", daemon_id=osd_id)
+        cmd_get_log_lines = (
+            f'awk \'$1 >= "{init_time}" && $1 <= "{end_time}"\' '
+            f"/var/log/ceph/{fsid}/ceph-osd.{osd_id}.log"
+        )
+        osd_log_lines, err = host.exec_command(
```

Comment:
Please re-direct the output to a file
file path should be accepted as a input from the user
file name can be generated on the fly

Reply:
Code modified as per the comments.